apps/sms/views/shared/js/startup.js

```diff
@@ -89,14 +89,15 @@ var Startup = exports.Startup = {
 
       // Init UI Managers
       TimeHeaders.init();
-      ConversationView.init();
-      MessagingClient.init(App.instanceId);
       MozMobileConnectionsClient.init(App.instanceId);
-      Information.initDefaultViews();
+      MessagingClient.init(App.instanceId).then(() => {
```

Comment:
Here I change the view init timing because the event is registered only when service is connected with app instance ID. Not sure if there is any better why to initialize the event listener because it must have longer gap between DOM event fired from iframe and view actually register the handler(at least it's async and anything could happen during this gap).